README.md

```diff
@@ -0,0 +1,16 @@
+#[React-Waypoint Homepage](http://brigade.github.io/react-waypoint/)
+
+##Adding Examples
+
+1. Add an inline script to the bottom of `index.html` using the script type
+   `text/jsx`. The in-browser JSX parser will parse your JSX. You must create a
+   component in the global namespace that renders your example.
+2. Add an `id` to your script tag that has the name of your example, lower
+   cased and with dashes instead of spaces, followed by "-example-script" . Example:
+   "Infinite Scroll" -> `infinite-scroll-example-script`
+3. Add the lower-cased and dashed name of your example to the array passed for the `exampleNames` prop to
```

Comment:
Probably won't revise the commit just to fix that, but next time I touch this file I can fix it.